shared/checks/oval/package_gnome-desktop3_installed.xml

```diff
@@ -0,0 +1,25 @@
+<def-group>
```

Comment:
This file should not exist and be generated with templates.

Reply:
In addition, nm-connection-editor is a dependency of gdm, so this actually should be package_gdm_installed.